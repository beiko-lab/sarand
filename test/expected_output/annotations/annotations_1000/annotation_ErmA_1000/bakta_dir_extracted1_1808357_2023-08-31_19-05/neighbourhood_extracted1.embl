ID   contig_1; ; linear; DNA; ; UNC; 2732 BP.
XX
AC   contig_1;
XX
DE   contig_1, whole genome shotgun sequence
XX
OS   .
OC   .
XX
CC   Annotated with Bakta
CC   Software: v1.8.1
CC   Database: v5.0, light
CC   DOI: 10.1099/mgen.0.000685
CC   URL: github.com/oschwengers/bakta
CC   
CC   ##Genome Annotation Summary:##
CC   Annotation Date                :: 08/31/2023, 19:05:40
CC   CDSs                           ::     2
CC   tRNAs                          ::     0
CC   tmRNAs                         ::     0
CC   rRNAs                          ::     0
CC   ncRNAs                         ::     0
CC   regulatory ncRNAs              ::     0
CC   CRISPR Arrays                  ::     0
CC   oriCs/oriVs                    ::     0
CC   oriTs                          ::     0
CC   gaps                           ::     0
CC   pseudogenes                    ::     0
XX
FH   Key             Location/Qualifiers
FH
FT   source          1..2732
FT                   /mol_type="genomic DNA"
FT   gene            1001..1732
FT                   /locus_tag="NFDPJF_00005"
FT                   /gene="erm(A)"
FT   CDS             1001..1732
FT                   /db_xref="NCBIProtein:WP_001072201.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P13956"
FT                   /product="23S rRNA (adenine(2058)-N(6))-methyltransferase
FT                   Erm(A)"
FT                   /locus_tag="NFDPJF_00005"
FT                   /protein_id="gnl|Bakta|NFDPJF_00005"
FT                   /translation="MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFT
FT                   KELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPY
FT                   NISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYF
FT                   HPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVT
FT                   NINKLSKEQFLSIFNSYKLFH"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P13956"
FT                   /gene="erm(A)"
FT   gene            complement(1858..2640)
FT                   /locus_tag="NFDPJF_00010"
FT                   /gene="ant(9)-Ia"
FT   CDS             complement(1858..2640)
FT                   /db_xref="NCBIProtein:WP_000067268.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P0A0D0"
FT                   /product="aminoglycoside nucleotidyltransferase ANT(9)-Ia"
FT                   /locus_tag="NFDPJF_00010"
FT                   /protein_id="gnl|Bakta|NFDPJF_00010"
FT                   /translation="MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRIN
FT                   SDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPK
FT                   REFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIR
FT                   RAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLL
FT                   DIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P0A0D0"
FT                   /gene="ant(9)-Ia"
XX
SQ   Sequence 2732 BP; 928 A; 502 C; 404 G; 898 T; 0 other;
     aaaaactctt ggattgcccc taatttataa acataggtag gcggtgtctc gacttcagat        60
     acattatcaa gaataactat tcttccttcc tcattcaaca gttccttcat ttgctgtatt       120
     acgctggcta tatcatccaa atgatgaaag gttgttcggc ttacaataaa atcaaacttc       180
     tcattaaaat taagttgttc tgcattcata ttcagataga ccgtatttgt tagttgacgt       240
     ttagatttgg caagatcgag catttgatta gaaatatcaa tccctaccac ttcatcataa       300
     taacttgcta atttctccac taacaaaccc gagccacatc cgatatctaa tgctctgcct       360
     ttctttggag acatattaga cacaaagaat gaataatcat tcaaaagctc attcacgaaa       420
     tcgtaatctt ctgcaacctt atcaaactgt gattctattg tattcaaaaa gatcccccat       480
     tcctacttta tcgacattct ttcattactt accactttag atgttttttc gttggggata       540
     aaacttccct ttagacaatt ttatccaaag acaatacaac agtgcaactt tattaaagtc       600
     actgtccttt atcgcagcct ttacttttta gtaaagacag tggcttctct tatcaagttt       660
     caaaacatat tattttgaag aaaacgtcca tctgaagtgt caagtgcaaa attacatata       720
     aaggtttatt ctaaaatgaa aagatgatac aatcatattc agttacataa ggaggtttca       780
     attatgtgca ccagtatcgc agtagtagaa attactttat ctcattcata atgaaaaaaa       840
     tggaaaggag ataaaagtat gggtactttt tctatatttg ttattaataa agttcgttat       900
     caaccaaatc aaaattaatt ggttataatg aacgcttaat gtcagttcat tataaccagt       960
     aaggagaagg ttataatgaa ccagtaagga gaaggttata atgaaccaga aaaaccctaa      1020
     agacacgcaa aattttatta cttctaaaaa gcatgtaaaa gaaatattga atcacacgaa      1080
     tatcagtaaa caagacaacg taatagaaat cggatcagga aaaggacatt ttaccaaaga      1140
     gctagtcaaa atgagtcgat cagttactgc tatagaaatt gatggaggct tatgtcaagt      1200
     gactaaagaa gcggtaaacc cctctgagaa tataaaagtg attcaaacgg atattctaaa      1260
     attttccttc ccaaaacata taaactataa gatatatggt aatattcctt ataacatcag      1320
     tacggatatt gtcaaaagaa ttacctttga aagtcaggct aaatatagct atcttatcgt      1380
     tgagaaggga tttgcgaaaa gattgcaaaa tctgcaacga gctttgggtt tactattaat      1440
     ggtggagatg gatataaaaa tgctcaaaaa agtaccacca ctatattttc atcctaagcc      1500
     aagtgtagac tctgtattga ttgttcttga acgacatcaa ccattgattt caaagaagga      1560
     ctacaaaaag tatcgatctt ttgtttataa gtgggtaaac cgtgaatatc gtgttctttt      1620
     cactaaaaac caattccgac aggctttgaa gcatgcaaat gtcactaata ttaataaact      1680
     atcgaaggaa caatttcttt ctattttcaa tagttacaaa ttgtttcact aaattaaagt      1740
     aataaagcgt tctctaattt cacaagagga cgctttattc ttcccaaaaa ttgttcaata      1800
     tttatcaata aatcagtagt tttaaaagta agcacctgtt attgcaataa aattagccta      1860
     attgagagaa gtttctatag aatttttcat atacttaacg agtgctttca cctttgaata      1920
     tagtccttcc cacttatcat cacactctcc ccgatagcct tttctagcta tatccagtaa      1980
     agttacatgc tctttaggta aaagaggtat agcccattct gcagcgacat ctttcgaggt      2040
     aatttcacca gtagtcactg tttgccacat tcgagctagg gttaaaatta cattacgctc      2100
     atcacctttt atcccctcaa ttagttctgg caaagaatcc ttaattgctc ttcgaatatc      2160
     tgtcaaaggt acggagacaa gtatacttga agaatcagga ccaaatagag aaatactatt      2220
     ctttcttgct tgtgctaaaa caatagccaa atcaggatca tagcttggtt cctgaatttg      2280
     tccattctca aattcacccc tgagccactc accgtatata aattctcttt ttggaggata      2340
     ttgccaaggg acaacttcac tcctatttat aaccgtaact tcaagtggtc taacagaatc      2400
     cgtatttcca atctttcctg atatagtcat tagtctttct gttagttttt ttcgagttaa      2460
     ttgaggtaaa ctatgattca cgacgactag aacatctaca tcgctgttaa tgcgtaaacc      2520
     accatttact gctgaaccaa atagatatac tccaactatt gaacttccaa ataaatcttt      2580
     tacgattttt aatgtttgaa tcgcttgatt tggtattttt ccgttaatca aattgctcat      2640
     gatttcacct cgttgattat gttcatataa agtttatatt gatactcaat ttacttaccc      2700
     tagattggac atatacttaa attactgttc aa                                    2732
//
