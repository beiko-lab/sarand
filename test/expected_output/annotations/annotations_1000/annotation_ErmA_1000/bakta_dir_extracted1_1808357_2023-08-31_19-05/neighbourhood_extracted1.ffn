>NFDPJF_00005 23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A)
ATGAACCAGAAAAACCCTAAAGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAAGAAATATTGAATCACACGAATATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGAAAAGGACATTTTACCAAAGAGCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATTGATGGAGGCTTATGTCAAGTGACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTGATTCAAACGGATATTCTAAAATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGTAATATTCCTTATAACATCAGTACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCTAAATATAGCTATCTTATCGTTGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGAGCTTTGGGTTTACTATTAATGGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCACTATATTTTCATCCTAAGCCAAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAACCATTGATTTCAAAGAAGGACTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAACCGTGAATATCGTGTTCTTTTCACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAATGTCACTAATATTAATAAACTATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAATTGTTTCACTAA
>NFDPJF_00010 aminoglycoside nucleotidyltransferase ANT(9)-Ia
ATGAGCAATTTGATTAACGGAAAAATACCAAATCAAGCGATTCAAACATTAAAAATCGTAAAAGATTTATTTGGAAGTTCAATAGTTGGAGTATATCTATTTGGTTCAGCAGTAAATGGTGGTTTACGCATTAACAGCGATGTAGATGTTCTAGTCGTCGTGAATCATAGTTTACCTCAATTAACTCGAAAAAAACTAACAGAAAGACTAATGACTATATCAGGAAAGATTGGAAATACGGATTCTGTTAGACCACTTGAAGTTACGGTTATAAATAGGAGTGAAGTTGTCCCTTGGCAATATCCTCCAAAAAGAGAATTTATATACGGTGAGTGGCTCAGGGGTGAATTTGAGAATGGACAAATTCAGGAACCAAGCTATGATCCTGATTTGGCTATTGTTTTAGCACAAGCAAGAAAGAATAGTATTTCTCTATTTGGTCCTGATTCTTCAAGTATACTTGTCTCCGTACCTTTGACAGATATTCGAAGAGCAATTAAGGATTCTTTGCCAGAACTAATTGAGGGGATAAAAGGTGATGAGCGTAATGTAATTTTAACCCTAGCTCGAATGTGGCAAACAGTGACTACTGGTGAAATTACCTCGAAAGATGTCGCTGCAGAATGGGCTATACCTCTTTTACCTAAAGAGCATGTAACTTTACTGGATATAGCTAGAAAAGGCTATCGGGGAGAGTGTGATGATAAGTGGGAAGGACTATATTCAAAGGTGAAAGCACTCGTTAAGTATATGAAAAATTCTATAGAAACTTCTCTCAATTAG
