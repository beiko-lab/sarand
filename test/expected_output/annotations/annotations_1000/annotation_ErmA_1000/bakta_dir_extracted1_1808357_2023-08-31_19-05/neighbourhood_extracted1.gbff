LOCUS       contig_1                2732 bp    DNA     linear   UNK 31-AUG-2023
DEFINITION  contig_1, whole genome shotgun sequence.
ACCESSION   contig_1
VERSION     contig_1
KEYWORDS    .
SOURCE      None
  ORGANISM  .
            .
COMMENT     Annotated with Bakta
            Software: v1.8.1
            Database: v5.0, light
            DOI: 10.1099/mgen.0.000685
            URL: github.com/oschwengers/bakta
            
            ##Genome Annotation Summary:##
            Annotation Date                :: 08/31/2023, 19:05:40
            CDSs                           ::     2
            tRNAs                          ::     0
            tmRNAs                         ::     0
            rRNAs                          ::     0
            ncRNAs                         ::     0
            regulatory ncRNAs              ::     0
            CRISPR Arrays                  ::     0
            oriCs/oriVs                    ::     0
            oriTs                          ::     0
            gaps                           ::     0
            pseudogenes                    ::     0
FEATURES             Location/Qualifiers
     source          1..2732
                     /mol_type="genomic DNA"
     gene            1001..1732
                     /locus_tag="NFDPJF_00005"
                     /gene="erm(A)"
     CDS             1001..1732
                     /db_xref="NCBIProtein:WP_001072201.1"
                     /db_xref="SO:0001217"
                     /db_xref="UniRef:UniRef50_P13956"
                     /product="23S rRNA (adenine(2058)-N(6))-methyltransferase
                     Erm(A)"
                     /locus_tag="NFDPJF_00005"
                     /protein_id="gnl|Bakta|NFDPJF_00005"
                     /translation="MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFT
                     KELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPY
                     NISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYF
                     HPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVT
                     NINKLSKEQFLSIFNSYKLFH"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniRef:UniRef50_P13956"
                     /gene="erm(A)"
     gene            complement(1858..2640)
                     /locus_tag="NFDPJF_00010"
                     /gene="ant(9)-Ia"
     CDS             complement(1858..2640)
                     /db_xref="NCBIProtein:WP_000067268.1"
                     /db_xref="SO:0001217"
                     /db_xref="UniRef:UniRef50_P0A0D0"
                     /product="aminoglycoside nucleotidyltransferase ANT(9)-Ia"
                     /locus_tag="NFDPJF_00010"
                     /protein_id="gnl|Bakta|NFDPJF_00010"
                     /translation="MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRIN
                     SDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPK
                     REFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIR
                     RAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLL
                     DIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniRef:UniRef50_P0A0D0"
                     /gene="ant(9)-Ia"
ORIGIN
        1 aaaaactctt ggattgcccc taatttataa acataggtag gcggtgtctc gacttcagat
       61 acattatcaa gaataactat tcttccttcc tcattcaaca gttccttcat ttgctgtatt
      121 acgctggcta tatcatccaa atgatgaaag gttgttcggc ttacaataaa atcaaacttc
      181 tcattaaaat taagttgttc tgcattcata ttcagataga ccgtatttgt tagttgacgt
      241 ttagatttgg caagatcgag catttgatta gaaatatcaa tccctaccac ttcatcataa
      301 taacttgcta atttctccac taacaaaccc gagccacatc cgatatctaa tgctctgcct
      361 ttctttggag acatattaga cacaaagaat gaataatcat tcaaaagctc attcacgaaa
      421 tcgtaatctt ctgcaacctt atcaaactgt gattctattg tattcaaaaa gatcccccat
      481 tcctacttta tcgacattct ttcattactt accactttag atgttttttc gttggggata
      541 aaacttccct ttagacaatt ttatccaaag acaatacaac agtgcaactt tattaaagtc
      601 actgtccttt atcgcagcct ttacttttta gtaaagacag tggcttctct tatcaagttt
      661 caaaacatat tattttgaag aaaacgtcca tctgaagtgt caagtgcaaa attacatata
      721 aaggtttatt ctaaaatgaa aagatgatac aatcatattc agttacataa ggaggtttca
      781 attatgtgca ccagtatcgc agtagtagaa attactttat ctcattcata atgaaaaaaa
      841 tggaaaggag ataaaagtat gggtactttt tctatatttg ttattaataa agttcgttat
      901 caaccaaatc aaaattaatt ggttataatg aacgcttaat gtcagttcat tataaccagt
      961 aaggagaagg ttataatgaa ccagtaagga gaaggttata atgaaccaga aaaaccctaa
     1021 agacacgcaa aattttatta cttctaaaaa gcatgtaaaa gaaatattga atcacacgaa
     1081 tatcagtaaa caagacaacg taatagaaat cggatcagga aaaggacatt ttaccaaaga
     1141 gctagtcaaa atgagtcgat cagttactgc tatagaaatt gatggaggct tatgtcaagt
     1201 gactaaagaa gcggtaaacc cctctgagaa tataaaagtg attcaaacgg atattctaaa
     1261 attttccttc ccaaaacata taaactataa gatatatggt aatattcctt ataacatcag
     1321 tacggatatt gtcaaaagaa ttacctttga aagtcaggct aaatatagct atcttatcgt
     1381 tgagaaggga tttgcgaaaa gattgcaaaa tctgcaacga gctttgggtt tactattaat
     1441 ggtggagatg gatataaaaa tgctcaaaaa agtaccacca ctatattttc atcctaagcc
     1501 aagtgtagac tctgtattga ttgttcttga acgacatcaa ccattgattt caaagaagga
     1561 ctacaaaaag tatcgatctt ttgtttataa gtgggtaaac cgtgaatatc gtgttctttt
     1621 cactaaaaac caattccgac aggctttgaa gcatgcaaat gtcactaata ttaataaact
     1681 atcgaaggaa caatttcttt ctattttcaa tagttacaaa ttgtttcact aaattaaagt
     1741 aataaagcgt tctctaattt cacaagagga cgctttattc ttcccaaaaa ttgttcaata
     1801 tttatcaata aatcagtagt tttaaaagta agcacctgtt attgcaataa aattagccta
     1861 attgagagaa gtttctatag aatttttcat atacttaacg agtgctttca cctttgaata
     1921 tagtccttcc cacttatcat cacactctcc ccgatagcct tttctagcta tatccagtaa
     1981 agttacatgc tctttaggta aaagaggtat agcccattct gcagcgacat ctttcgaggt
     2041 aatttcacca gtagtcactg tttgccacat tcgagctagg gttaaaatta cattacgctc
     2101 atcacctttt atcccctcaa ttagttctgg caaagaatcc ttaattgctc ttcgaatatc
     2161 tgtcaaaggt acggagacaa gtatacttga agaatcagga ccaaatagag aaatactatt
     2221 ctttcttgct tgtgctaaaa caatagccaa atcaggatca tagcttggtt cctgaatttg
     2281 tccattctca aattcacccc tgagccactc accgtatata aattctcttt ttggaggata
     2341 ttgccaaggg acaacttcac tcctatttat aaccgtaact tcaagtggtc taacagaatc
     2401 cgtatttcca atctttcctg atatagtcat tagtctttct gttagttttt ttcgagttaa
     2461 ttgaggtaaa ctatgattca cgacgactag aacatctaca tcgctgttaa tgcgtaaacc
     2521 accatttact gctgaaccaa atagatatac tccaactatt gaacttccaa ataaatcttt
     2581 tacgattttt aatgtttgaa tcgcttgatt tggtattttt ccgttaatca aattgctcat
     2641 gatttcacct cgttgattat gttcatataa agtttatatt gatactcaat ttacttaccc
     2701 tagattggac atatacttaa attactgttc aa
//
