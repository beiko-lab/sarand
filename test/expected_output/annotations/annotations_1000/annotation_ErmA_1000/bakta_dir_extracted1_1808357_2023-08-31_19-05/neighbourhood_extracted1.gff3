##gff-version 3
##feature-ontology https://github.com/The-Sequence-Ontology/SO-Ontologies/blob/v3.1/so.obo
# Annotated with Bakta
# Software: v1.8.1
# Database: v5.0, light
# DOI: 10.1099/mgen.0.000685
# URL: github.com/oschwengers/bakta
##sequence-region contig_1 1 2732
contig_1	Bakta	region	1	2732	.	+	.	ID=contig_1;Name=contig_1
contig_1	Prodigal	CDS	1001	1732	.	+	0	ID=NFDPJF_00005;Name=23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A);locus_tag=NFDPJF_00005;product=23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A);Dbxref=NCBIProtein:WP_001072201.1,SO:0001217,UniRef:UniRef50_P13956;gene=erm(A)
contig_1	Prodigal	CDS	1858	2640	.	-	0	ID=NFDPJF_00010;Name=aminoglycoside nucleotidyltransferase ANT(9)-Ia;locus_tag=NFDPJF_00010;product=aminoglycoside nucleotidyltransferase ANT(9)-Ia;Dbxref=NCBIProtein:WP_000067268.1,SO:0001217,UniRef:UniRef50_P0A0D0;gene=ant(9)-Ia
##FASTA
>contig_1
AAAAACTCTTGGATTGCCCCTAATTTATAAACATAGGTAGGCGGTGTCTCGACTTCAGAT
ACATTATCAAGAATAACTATTCTTCCTTCCTCATTCAACAGTTCCTTCATTTGCTGTATT
ACGCTGGCTATATCATCCAAATGATGAAAGGTTGTTCGGCTTACAATAAAATCAAACTTC
TCATTAAAATTAAGTTGTTCTGCATTCATATTCAGATAGACCGTATTTGTTAGTTGACGT
TTAGATTTGGCAAGATCGAGCATTTGATTAGAAATATCAATCCCTACCACTTCATCATAA
TAACTTGCTAATTTCTCCACTAACAAACCCGAGCCACATCCGATATCTAATGCTCTGCCT
TTCTTTGGAGACATATTAGACACAAAGAATGAATAATCATTCAAAAGCTCATTCACGAAA
TCGTAATCTTCTGCAACCTTATCAAACTGTGATTCTATTGTATTCAAAAAGATCCCCCAT
TCCTACTTTATCGACATTCTTTCATTACTTACCACTTTAGATGTTTTTTCGTTGGGGATA
AAACTTCCCTTTAGACAATTTTATCCAAAGACAATACAACAGTGCAACTTTATTAAAGTC
ACTGTCCTTTATCGCAGCCTTTACTTTTTAGTAAAGACAGTGGCTTCTCTTATCAAGTTT
CAAAACATATTATTTTGAAGAAAACGTCCATCTGAAGTGTCAAGTGCAAAATTACATATA
AAGGTTTATTCTAAAATGAAAAGATGATACAATCATATTCAGTTACATAAGGAGGTTTCA
ATTATGTGCACCAGTATCGCAGTAGTAGAAATTACTTTATCTCATTCATAATGAAAAAAA
TGGAAAGGAGATAAAAGTATGGGTACTTTTTCTATATTTGTTATTAATAAAGTTCGTTAT
CAACCAAATCAAAATTAATTGGTTATAATGAACGCTTAATGTCAGTTCATTATAACCAGT
AAGGAGAAGGTTATAATGAACCAGTAAGGAGAAGGTTATAATGAACCAGAAAAACCCTAA
AGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAAGAAATATTGAATCACACGAA
TATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGAAAAGGACATTTTACCAAAGA
GCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATTGATGGAGGCTTATGTCAAGT
GACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTGATTCAAACGGATATTCTAAA
ATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGTAATATTCCTTATAACATCAG
TACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCTAAATATAGCTATCTTATCGT
TGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGAGCTTTGGGTTTACTATTAAT
GGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCACTATATTTTCATCCTAAGCC
AAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAACCATTGATTTCAAAGAAGGA
CTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAACCGTGAATATCGTGTTCTTTT
CACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAATGTCACTAATATTAATAAACT
ATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAATTGTTTCACTAAATTAAAGT
AATAAAGCGTTCTCTAATTTCACAAGAGGACGCTTTATTCTTCCCAAAAATTGTTCAATA
TTTATCAATAAATCAGTAGTTTTAAAAGTAAGCACCTGTTATTGCAATAAAATTAGCCTA
ATTGAGAGAAGTTTCTATAGAATTTTTCATATACTTAACGAGTGCTTTCACCTTTGAATA
TAGTCCTTCCCACTTATCATCACACTCTCCCCGATAGCCTTTTCTAGCTATATCCAGTAA
AGTTACATGCTCTTTAGGTAAAAGAGGTATAGCCCATTCTGCAGCGACATCTTTCGAGGT
AATTTCACCAGTAGTCACTGTTTGCCACATTCGAGCTAGGGTTAAAATTACATTACGCTC
ATCACCTTTTATCCCCTCAATTAGTTCTGGCAAAGAATCCTTAATTGCTCTTCGAATATC
TGTCAAAGGTACGGAGACAAGTATACTTGAAGAATCAGGACCAAATAGAGAAATACTATT
CTTTCTTGCTTGTGCTAAAACAATAGCCAAATCAGGATCATAGCTTGGTTCCTGAATTTG
TCCATTCTCAAATTCACCCCTGAGCCACTCACCGTATATAAATTCTCTTTTTGGAGGATA
TTGCCAAGGGACAACTTCACTCCTATTTATAACCGTAACTTCAAGTGGTCTAACAGAATC
CGTATTTCCAATCTTTCCTGATATAGTCATTAGTCTTTCTGTTAGTTTTTTTCGAGTTAA
TTGAGGTAAACTATGATTCACGACGACTAGAACATCTACATCGCTGTTAATGCGTAAACC
ACCATTTACTGCTGAACCAAATAGATATACTCCAACTATTGAACTTCCAAATAAATCTTT
TACGATTTTTAATGTTTGAATCGCTTGATTTGGTATTTTTCCGTTAATCAAATTGCTCAT
GATTTCACCTCGTTGATTATGTTCATATAAAGTTTATATTGATACTCAATTTACTTACCC
TAGATTGGACATATACTTAAATTACTGTTCAA
