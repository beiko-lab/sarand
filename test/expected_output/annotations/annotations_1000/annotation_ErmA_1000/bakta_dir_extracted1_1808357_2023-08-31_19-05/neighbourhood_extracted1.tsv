# Annotated with Bakta
# Software: v1.8.1
# Database: v5.0, light
# DOI: 10.1099/mgen.0.000685
# URL: github.com/oschwengers/bakta
#Sequence Id	Type	Start	Stop	Strand	Locus Tag	Gene	Product	DbXrefs
contig_1	cds	1001	1732	+	NFDPJF_00005	erm(A)	23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A)	NCBIProtein:WP_001072201.1, SO:0001217, UniRef:UniRef50_P13956
contig_1	cds	1858	2640	-	NFDPJF_00010	ant(9)-Ia	aminoglycoside nucleotidyltransferase ANT(9)-Ia	NCBIProtein:WP_000067268.1, SO:0001217, UniRef:UniRef50_P0A0D0
